mmv1/third_party/terraform/services/bigquery/resource_bigquery_table.go.erb

```diff
@@ -1597,10 +1674,46 @@ func resourceBigQueryTableRead(d *schema.ResourceData, meta interface{}) error {
 		}
 	}
 
+	<% unless version == 'ga' -%>
+	// Get API fields for TableReplicationInfo not available in the client library yet.
+	url, err := tpgresource.ReplaceVars(d, config, "{{BigQueryBasePath}}projects/{{project}}/datasets/{{dataset_id}}/tables/{{table_id}}")
```

Comment:
Oh interesting, do you know when the client library will have this available? Mainly I'm just wondering if we need some way to remember to come back to this later.

Reply:
Some time in Q1 but not in time for the feature launch. I was advised this way of doing the read during Google TF office hour on 12/5/2023 (the OH doc should still have the notes).

Reply:
Ok, sounds good. I think it might be worth adding a `TODO:` prefix to the comment if this is something you expect to be revisited in the near-term, just so that the different pattern is clearly highlighted as something that is temporary.